packages/passport/sdk/src/errors/passportError.ts

```diff
@@ -52,6 +53,9 @@ export const withPassportError = async <T>(
       errorMessage = (error as Error).message;
     }
 
-    throw new PassportError(errorMessage, customErrorType);
+    const passportError = new PassportError(errorMessage, customErrorType);
+    trackError('passport', customErrorType, passportError);
```

Comment:
I don't believe we should be using `customErrorType` as the second argument. `trackError` accepts a `eventName` as the second argument, and it also prepends `trackError_` to the `eventName`, so in the context of transfer this should probably be something like `imxTransfer` to produce an event called `trackError_imxTransfer`

Reply:
Yes, shows up like this in the query results: `metrics_eventName:trackErrorTRANSFERERROR`. I'll give it a better event name